server.js

```diff
@@ -1,11 +1,137 @@
 'use strict';
 
-const { ApolloServer } = require('apollo-server');
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-console */
+
+require('dotenv').config();
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const session = require('express-session');
+const passport = require('passport');
+const saml = require('passport-saml');
+const { ApolloServer } = require('apollo-server-express');
+const { Op } = require('sequelize');
+const db = require('./models');
 const { schema } = require('./graphql');
 
-const server = new ApolloServer({ schema });
+const PORT = 4000;
+const HOST = `http://localhost:${PORT}`;
+const SESSION_SECRET = 'notsecure';
```

Comment:
this is just for testing now right? if so can we add a todo on the line above to change this?

Reply:
oh yeah good catch, will do